Tests/Svg.UnitTests/ImageComparisonTest.cs

```diff
@@ -62,8 +72,10 @@ public void CompareSvgImageWithReference(ImageTestDataSource.TestData testData)
             CompareSvgImageWithReferenceImpl(baseName, svgPath, pngPath, testSaveLoad);
         }
 #endif
+        
+        
 
-        private void CompareSvgImageWithReferenceImpl(string baseName,
+        public static void CompareSvgImageWithReferenceImpl(string baseName,

```

Comment:
Please revert this line.

Reply:
This is used in the other test, so it cannot be private. Can be made internal instead.